automation_tools/__init__.py

```diff
@@ -914,11 +914,11 @@ def product_install(distribution, create_vm=False, certificate_url=None,
     distribution = distribution.lower()
 
     install_tasks = {
-        'cdn': cdn_install,
-        'downstream': downstream_install,
-        'iso': iso_install,
-        'upstream': upstream_install,
         'sam-upstream': sam_upstream_install,
+        'satellite6-cdn': cdn_install,
+        'satellite6-downstream': downstream_install,
+        'satellite6-iso': iso_install,
+        'satellite6-upstream': upstream_install,
```

Comment:
I say this because I rely heavily on those when setting up systems using automation-tools. I'm your audience of one. :)